src/renderer/components/+custom-resources/crd-resource-details.tsx

```diff
@@ -60,24 +60,15 @@ function convertSpecValue(value: any): any {
 
 @observer
 export class CrdResourceDetails extends React.Component<Props> {
-  constructor(props: Props) {
-    super(props);
-    makeObservable(this);
-  }
-
-  @computed get crd() {
-    return crdStore.getByObject(this.props.object);
-  }
-
-  renderAdditionalColumns(crd: CustomResourceDefinition, columns: AdditionalPrinterColumnsV1[]) {
+  renderAdditionalColumns(resource: KubeObject, columns: AdditionalPrinterColumnsV1[]) {
```

Comment:
This component was incorrect because it said it was dealing with `CustomResourceDefenition`'s as its main type. It is not, it is dealing with custom resources (which are from Lens' perspective, just `KubeObject`'s).